guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,705 @@
 {
```

Comment:
<p>More new models to be listed:</p><ul><li>MultiTaskRoadExtractor</li><li>ChangeDetector</li><li>Pix2Pix</li></ul><p>Text models:</p><ul><li>TextClassifier</li><li>SequenceToSequence</li><li>Inference Only Models</li><li class="ql-indent-1">FillMask</li><li class="ql-indent-1">QuestionAnswering</li><li class="ql-indent-1">TextGenerator</li><li class="ql-indent-1">TextSummarizer</li><li class="ql-indent-1">TextTranslator</li><li class="ql-indent-1">ZeroShotClassifier</li></ul><p> </p><p>Adds ability to show intermediate results on Tensorboard during training for:</p><ul><li><code>FeatureClassifier</code></li><li><code>Single Shot Detector</code></li><li><code>Retinanet</code></li><li><code>UNetClassifier</code></li><li><code>SuperResolution</code></li><li><code>FasterRCNN</code></li><li><code>YOLOv3</code></li><li><code>MaskRCNN</code></li><li>&nbsp;<code>PSPNetClassifier</code></li><li><code>&nbsp;DeepLab</code></li><li><code>&nbsp;BDCNEdgeDetector</code></li><li><code>HEDEdgeDetector</code></li><li><code>MultiTaskRoadExtractor</code></li></ul><p> </p><p>Adds&nbsp;<code><a href="https://github.com/facebookresearch/detectron2/tree/master/projects/PointRend" target="_blank">PointRend</a></code>&nbsp;to improve performance for:</p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#deeplab" target="_blank">DeepLab</a></code></li><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#pspnetclassifier" target="_blank">PSPNetClassifer</a></code></li><li><code>MaskRCNN</code></li></ul><p> </p><p>SuperResolution</p><ul><li>export_tiles support in Pro for creating training data </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Done!